unity-client/Assets/Scripts/MainScripts/DCL/Environment/PlatformContext/PlatformContext.cs

```diff
@@ -14,20 +14,23 @@ public class PlatformContext : System.IDisposable
         public readonly IClipboard clipboard;
         public readonly IPhysicsSyncController physicsSyncController;
         public readonly IDebugController debugController;
+        public readonly IWebRequestController webRequestController;
```

Comment:
This should generate the following syntax that I think is a bit inconvenient:

`Environment.i.platform.webRequestController.webRequestAB.Get();`

I think it can be improved to one of the following:

`Environment.i.platform.fetch.assetBundles.Get();`

`Environment.i.platform.request.assetBundle.Get();`

`Environment.i.platform.webRequest.assetBundle.Get();`

you get the drift.

Reply:
I like you suggestion. Finally I have decided to follow the next way:
`Environment.i.platform.webRequest.generic.Get()`
`Environment.i.platform.webRequest.assetBundle.Get()`
`Environment.i.platform.webRequest.texture.Get()`
`Environment.i.platform.webRequest.audio.Get()`